### PR TITLE
Stop using "checksumming" instead of "encrypting" in the docs

### PR DESCRIPTION
**Pull request by vice**
All encryption systems have a formal inverse function to decrypt
A hash is a one way function without inverse by definition

##### SUMMARY
Change title in template filter documentation that says encrypting when hashing is described

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
Template hashing and checksum filters

### CODE REVIEW

**Review by webknjaz — CHANGES_REQUESTED** [review]
Thanks for your contribution, I reviewed it and discovered that below the hashing examples in this documentation section, there are also examples of encryption with `vault`/`unvault` at the bottom. So it sounds like there's both encryption and hashing present. Note that the title also mentions "checksumming" — I believe that this part is what refers to hashing in the examples. Would you agree with this judgement?

File: docs/docsite/rst/user_guide/playbooks_filters.rst [hunk 1]
```diff
@@ -1518,8 +1518,8 @@ This allows for dynamic generation of VLAN lists on a Cisco IOS tagged interface
 
 .. _hash_filters:
 
-Encrypting and checksumming strings and passwords
-=================================================
+Hashing and checksumming strings and passwords
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_filters.rst]:
```suggestion
Hashing and encrypting strings and passwords
```

Comment (samccann) [reply docs/docsite/rst/user_guide/playbooks_filters.rst]: Most of what is on that page is hashing, with one checksum example, and the new encrypting section so I think this title covers it well enough.

**Comment by samccann** [comment]
@webknjaz what do you think of the updated heading now?

**Review by webknjaz — APPROVED** [review]
LGTM

**Pull request merged**